### DISCUSSION TITLE
[gatsby-source-filesystem] query allFile return empty

### DISCUSSION THREAD

**Discussion by kimthanh932000** [gatsbyjs/gatsby#32160] [Help] [answered]
I want to get all images stored in `images` folder but it always returns nothing when executing `allFile` query in graphQL playground. Did I do something wrong?

Screenshots below show my project settings.

![Screenshot from 2021-06-29 20-50-17](https://user-images.githubusercontent.com/30076762/123811112-29976a00-d91d-11eb-9b0a-3b2de177c263.png)
![Screenshot from 2021-06-29 20-49-36](https://user-images.githubusercontent.com/30076762/123811121-2bf9c400-d91d-11eb-8cb5-eb4e94a1fa52.png)
![Screenshot from 2021-06-29 20-49-15](https://user-images.githubusercontent.com/30076762/123811131-2d2af100-d91d-11eb-9db8-dd5cf5644607.png)

**Comment by kimthanh932000** [comment, marked as answer]
I fixed the problem. It was lacking `condition: true` for the plugin in `gatsby-config.js`.

**Comment by wardpeet** [comment]
`condition: true` is something you added yourself to filter, correct?

**Reply by kimthanh932000** [reply]
Yeah, it was added for filtering by another team member so I did not notice that.
